mods/wlssb/statuses.js

```diff
@@ -158,16 +158,18 @@ exports.BattleStatuses = {
 			}
 		},
 	},
-	ssbn640: {
+	lycaniumz: {
 		exists: true,
-		onStart: function (pokemon) {
-			this.add('c', '%SSBN-640', 'hey, im just here to shitpost or whatever.');
+		onStart: function (source) {
+			this.add('c', '%Lycanium Z', 'Heyyyyyyyyyyyyyyyyyyyyyy');
+			source.types = ["Fighting", "Ghost"];
```

Comment:
Forcing the foes ability to normalize and making yourself part ghost is too OP.

Reply:
Ok, PM'd u on discord

Reply:
I guess thats ok